R/rnet_group.R

```diff
@@ -1,5 +1,17 @@
 #' Assign segments in a route network to groups
 #'
+#' This function assigns linestring features, with together are assumed to
```

Comment:
Hi! Not 100% sure(and I'm not an English speaker) but I think it should be "*which* together...", no?